exportoptionsgenerator/exportoptionsgenerator.go

```diff
@@ -376,10 +366,10 @@ func addDistributionBundleIdentifierFromXcode12(exportOpts exportoptions.ExportO
 	return nil
 }
 
-func disableManagedBuildNumberFromXcode13(exportOpts exportoptions.ExportOptions) exportoptions.ExportOptions {
+func addManagedBuildNumberFromXcode13(exportOpts exportoptions.ExportOptions, isManageAppVersion bool) exportoptions.ExportOptions {
```

Comment:
Changed to support existing input (manage_version_and_build_number) in export-xcarchive.